src/main/java/org/springframework/integration/aws/inbound/kinesis/KinesisMessageDrivenChannelAdapter.java

```diff
@@ -245,6 +250,7 @@ public void setCheckpointMode(CheckpointMode checkpointMode) {
 
 	/**
 	 * Sets the interval between 2 checkpoints. Only used when checkpointMode is periodic.
+	 *
```

Comment:
No blank lines in method JavaDoc, please: https://github.com/spring-projects/spring-framework/wiki/Code-Style#javadoc-formatting

Reply:
Done.